spec/controllers/index_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative '../spec_helper'
+
+describe 'Index Controller' do
+
+  it 'should get the /sealions route' do
+    get '/sealions'
+    expect(last_response).to be_ok
+    expect(last_response.body).to include('ARK ARK ARK')
+  end
+
+  it 'should get the /sealions/:name route' do
+    get '/sealions/:name?name=6'
+    expect(last_response).to be_ok
+    expect(last_response.body).to include("Your name is 6")
+  end
+
+  it 'should redirect post to /sealions' do
+    post '/sealions'
+    expect(last_response).to be_redirect
+    follow_redirect!
+    expect(last_response).to be_ok
+    expect(last_response.body).to include('ARK ARK ARK!')
+  end
+
+  it 'should redirect put to /sealions' do
+    put '/sealions/:name'
```

Comment:
What is the :name? param here? Do you have a route that matches this?  the put should look something like '/sealions/#some_id', params: {name: ='zack'}